ilastik/applets/pixelClassification/pixelClassificationSerializer.py

```diff
@@ -29,12 +32,81 @@
     SerialClassifierFactorySlot,
     SerialPickleableSlot,
 )
+from lazyflow.slot import OutputSlot
+from typing import List, Tuple
+from ndstructs import Array5D, Slice5D
 
 import logging
 
 logger = logging.getLogger(__name__)
 
 
+class BackwardsCompatibleLabelSerialBlockSlot(SerialBlockSlot):
+    def get_corresponding_input_image_slot(self, labelSlot: OutputSlot) -> OutputSlot:
+        labelMultislot = labelSlot.operator.inputs[labelSlot.name]
```

Comment:
shouldn't `labelSlot.subindex` do the trick?

Reply:
There's even some logic inside `Slot.__getitem__` to handle tuple-indices. I'm impressed =D